v63002/inref/gtm5178.m

```diff
@@ -0,0 +1,16 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	;
+; All rights reserved.						;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm5178
+	for i=1:1:1 do
+	. . write "2 dots beneath a line with no dots",!
```

Comment:
Would be nice to have a few more combinations.

1) Line without for followed by a double dotted line.
2) Line without do followed by a single dotted line.
3) Line with do followed by a single dotted line followed by a double dotted line.
4) Line with do followed by a single dotted line with a do followed by a 2 dotted line with a do followed by a 4 dotted line followed by a 2 dotted line. Only the 4 dotted line should trigger a warning but the other lines should execute as intended so would be good to have a write command in each line (for each of the above combinations) so we know from the output which lines actually got executed.